Source/Flow/Public/Nodes/FlowNodeBase.h

```diff
@@ -28,7 +28,7 @@ typedef TFunction<void(UFlowNodeAddOn&)> FFlowNodeAddOnFunction;
 /**
  * The base class for UFlowNode and UFlowNodeAddOn, with their shared functionality
  */
-UCLASS(Abstract, HideCategories = Object)
+UCLASS(Abstract, BlueprintType, HideCategories = Object)
```

Comment:
Must be BlueprintType if UFlowNodeBase will serve the IFlowNativeExecutableInterface role